### ISSUE TITLE
package images failing to load

### ISSUE THREAD

**Issue by snopeks** [npm/marky-markdown#320]
We got a report that this logo image on the `kaiser-crawler` package page is not loading: 
https://www.npmjs.com/package/kaiser-crawler

<img width="669" alt="screen shot 2017-01-03 at 11 49 43 am" src="https://cloud.githubusercontent.com/assets/6379200/21621107/70050830-d1ac-11e6-914d-986812e5aa4c.png">

@ashleygwilliams suggested this was a bug in marky and could (probably) be fixed shortly, thanks for looking!

**Comment by revin** [comment]
Thanks for the report! Yep, this is indeed a marky bug: the `packagize` thing currently only works on markdown and doesn't drill into inline HTML to do GitHub URL rewriting, but the first few lines of this README are:
```markdown
# Kaiser - Node.js web crawler

<img src="logo/kaiser.png" alt="Kaiser Logo" width="222" height="220" />

[![NPM](https://nodei.co/npm/kaiser-crawler.png?downloads=true&downloadRank=true&stars=true)](https://nodei.co/npm/kaiser-crawler/)
```

**Comment by revin** [comment]
oops I'm sorry, not `packagize`, I always get the named confused. The plugin in question is `lib/plugin/github.js`

**Comment by revin** [comment]
I'm being Chatty McChatterton here with the responses, but further investigation has revealed that we _do_ drill into inline HTML (I forgot this was already fixed! xref #304 + #308). The cause of this particular bug is that the HTML isn't inline; it's in an `html_block` token. So we need to extend the plugin further to dig into those as well.
